diff -Naur kmod-34.2/debian/changelog kmod-34.2_new/debian/changelog
--- kmod-34.2/debian/changelog	2025-04-25 12:50:41.000000000 +0300
+++ kmod-34.2_new/debian/changelog	2025-11-16 23:25:34.103792800 +0200
@@ -1,3 +1,9 @@
+kmod (34.2-3000) unstable; urgency=medium
+
+  * Add zlib support
+
+ -- Denys Fedoryshchenko <denys.f@collabora.com>  Fri, 16 Nov 2025 11:37:04 +0300
+
 kmod (34.2-2) unstable; urgency=medium
 
   * kmod.initramfs-hook: if compressed modules have been copied to the
diff -Naur kmod-34.2/debian/control kmod-34.2_new/debian/control
--- kmod-34.2/debian/control	2025-04-25 12:47:10.000000000 +0300
+++ kmod-34.2_new/debian/control	2025-11-16 23:25:57.571523159 +0200
@@ -9,6 +9,7 @@
   liblzma-dev (>= 5.4.5-0.2),
   libssl-dev,
   libzstd-dev, zstd <!nocheck>,
+  zlib1g-dev,
   linux-headers-generic [!armel !i386 !mips64el !sh4 !x32] <!nocheck>,
   linux-headers-mips64r6el [mips64el] <!nocheck>,
   linux-headers-rpi [armel] <!nocheck>,
diff -Naur kmod-34.2/debian/rules kmod-34.2_new/debian/rules
--- kmod-34.2/debian/rules	2025-04-25 12:47:10.000000000 +0300
+++ kmod-34.2_new/debian/rules	2025-11-16 23:27:04.158101565 +0200
@@ -27,13 +27,13 @@
   -D docs=$(KMOD_ENABLE_DOCS) \
   -D openssl=enabled \
   -D dlopen=all \
-  -D zlib=disabled \
+  -D zlib=enabled \
   -D xz=enabled \
   -D zstd=enabled
 CONFFLAGS_udeb = $(CONFFLAGS) \
   -D docs=false \
   -D openssl=disabled \
-  -D zlib=disabled \
+  -D zlib=enabled \
   -D xz=enabled \
   -D zstd=disabled
 
