diff -Naur kmod-30+20221128/debian/changelog kmod-30+20221128-kernelci/debian/changelog
--- kmod-30+20221128/debian/changelog	2022-12-10 00:02:24.000000000 +0000
+++ kmod-30+20221128-kernelci/debian/changelog	2024-02-22 07:41:04.167204352 +0000
@@ -1,3 +1,9 @@
+kmod (30+20221128-2) unstable; urgency=medium
+  
+  * Add zlib support
+
+ -- Denys Fedoryshchenko <denys.f@collabora.com>  Fri, 22 Feb 2023 11:37:04 +0300
+
 kmod (30+20221128-1) unstable; urgency=medium
 
   * New upstream snapshot.
diff -Naur kmod-30+20221128/debian/control kmod-30+20221128-kernelci/debian/control
--- kmod-30+20221128/debian/control	2022-12-09 23:51:42.000000000 +0000
+++ kmod-30+20221128-kernelci/debian/control	2024-02-22 07:42:05.239717578 +0000
@@ -8,6 +8,7 @@
   liblzma-dev,
   libssl-dev,
   libzstd-dev, zstd <!nocheck>,
+  zlib1g-dev,
   gtk-doc-tools, xsltproc,
 Standards-Version: 4.6.1.0
 Rules-Requires-Root: no
diff -Naur kmod-30+20221128/debian/rules kmod-30+20221128-kernelci/debian/rules
--- kmod-30+20221128/debian/rules	2022-12-09 22:58:43.000000000 +0000
+++ kmod-30+20221128-kernelci/debian/rules	2024-02-22 07:38:04.049690690 +0000
@@ -21,6 +21,7 @@
   --enable-gtk-doc \
   --with-openssl \
   --with-xz \
+  --with-zlib \
   --with-zstd \
   --disable-test-modules \
   --enable-debug
